src/shell/atuin.fish

```diff
@@ -9,7 +7,8 @@ end
 function _atuin_postexec --on-event fish_postexec
     set s $status
     if test -n "$ATUIN_HISTORY_ID"
-        RUST_LOG=error atuin history end $ATUIN_HISTORY_ID --exit $s &; disown
+        RUST_LOG=error atuin history end $ATUIN_HISTORY_ID --exit $s &
+        disown
```

Comment:
these lines and 24-26 just got auto-formatted by `fish_indent` (a formatter for fish scripts). Let me know if you'd like to revert these lines.